From fb3b240c55c29d6c8bf8d0b8559a8a05cc963aa3 Mon Sep 17 00:00:00 2001
Message-ID: <fb3b240c55c29d6c8bf8d0b8559a8a05cc963aa3.1694125610.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1694125609.git.sharlatanus@gmail.com>
References: <cover.1694125609.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 7 Sep 2023 21:06:28 +0100
Subject: [PATCH 11/19] gnu: python-jwst: Update to 1.11.4.

* gnu/packages/astronomy.scm (python-jwst): Update to 1.11.4.
[propagated-inputs]: Add python-asdf-transform-schemas, python-jplephem.
[native-inputs]: Remove python-codecov, python-flake8.
---
 gnu/packages/astronomy.scm | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 2064a05c64..2c8e16f550 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -3181,13 +3181,13 @@ (define-public python-jplephem
 (define-public python-jwst
   (package
     (name "python-jwst")
-    (version "1.10.2")
+    (version "1.11.4")
     (source (origin
               (method url-fetch)
               (uri (pypi-uri "jwst" version))
               (sha256
                (base32
-                "1lmfyw2y7c84rs9xqavah9aidj478ijiiijlz6fag11xqn1vs98y"))))
+                "1q3kpdjr8j1y8hkpidy80aqxsp3k13sz4j5aagq3wnzvpnds3x73"))))
     (build-system pyproject-build-system)
     (arguments
      (list
@@ -3204,11 +3204,13 @@ (define-public python-jwst
     (propagated-inputs (list opencv
                              python-asdf
                              python-asdf-astropy
+                             python-asdf-transform-schemas
                              python-astropy
                              python-bayesicfitting
                              python-crds
                              python-drizzle
                              python-gwcs
+                             python-jplephem
                              python-jsonschema
                              python-numpy
                              python-photutils
@@ -3226,9 +3228,7 @@ (define-public python-jwst
                              python-stsci-imagestats
                              python-tweakwcs
                              python-wiimatch))
-    (native-inputs (list python-codecov
-                         python-colorama
-                         python-flake8
+    (native-inputs (list python-colorama
                          python-pytest
                          python-pytest-cov
                          python-pytest-doctestplus
-- 
2.41.0

